File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task147_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the array of integers (space separated): ");
        String[] input = scanner.nextLine().split(" ");
        int[] nums = new int[input.length];
        for (int i = 0; i < input.length; i++) {
            nums[i] = Integer.parseInt(input[i]);
        }
        System.out.print("Enter the value of k: ");
        int k = scanner.nextInt();
        Arrays.sort(nums);
        System.out.println("The kth largest element is: " + nums[nums.length - k]);
    }
}

File: dataset/output/PERPLEXITY_llama-3-sonar-large-32k-chat/Task147_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int find_kth_largest(int nums[], int n, int k) {
    sort(nums, nums + n);
    return nums[n - k];
}

int main() {
    int n, k;
    cout << "Enter the number of elements: ";
    cin >> n;
    int nums[n];
    cout << "Enter the array of integers: ";
    for (int i = 0; i < n; i++) {
        cin >> nums[i];
    }
    cout << "Enter the value of k: ";
    cin >> k;
    cout << "The kth largest element is: " << find_kth_largest(nums, n, k) << endl;
    return 0;
}